tests/test_pptx/test_pptx_integration.py

```
@@ -0,0 +1,491 @@
+            # Duplicate first slide
+            first_slide = presentation.slides[0]
+            try:
+                duplicated_slide = first_slide.duplicate(api_client)
+                assert isinstance(duplicated_slide, PowerPointSlide)
+
+                # Note: The duplicated slide might not automatically be added to the presentation
+                # depending on the implementation, so we might not see an increase in slide count
+                print(f"Original slides: {original_slide_count}")
+                print(f"Duplicated slide created: {duplicated_slide.objectId}")
+
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)